packages/d3fc-webgl/src/series/glLine.js

```diff
@@ -59,12 +70,26 @@ export default () => {
         }
     );
 
-    const program = programBuilder()
-        .mode(drawModes.TRIANGLES)
-        .verticesPerElement(12);
+    const elementIndices = elementIndicesBuilder().data([
```

Comment:
I know it's not part of this change technically but could we make use of `elementIndices` to remove the duplication in corner?

Reply:
Yes, I'll put that in.